packages/data/lib/di/injector_data.dart

```diff
@@ -72,9 +84,17 @@ void _initModuleRepository() {
 }
 
 void _initModuleInterceptor() {
-  GetIt.instance.registerSingleton<HeaderInterceptorTraktApi>(
-    HeaderInterceptorTraktApi(),
-  );
+  final environmentConfiguration = EnvironmentConfiguration();
+  if (environmentConfiguration.prod) {
+    GetIt.instance.registerSingleton<HeaderInterceptorTraktApi>(
+      HeaderInterceptorTraktApi(),
+    );
+  }
+  if (environmentConfiguration.sandbox) {
+    GetIt.instance.registerSingleton<SandboxHeaderInterceptorTraktApi>(
+      SandboxHeaderInterceptorTraktApi(),
+    );
+  }
```

Comment:
is there no chances to make only one interceptor for 2 flavors?

Reply:
yes, of course, but if there are more than 2?